### PR TITLE
Added finer control over :meth:`.Scene.wait` being static (i.e., no updaters) or not

### PR DESCRIPTION
**Pull request by behackl**
## Overview: What does this pull request change?

- Added keyword argument `frozen_frame` to `Wait` and `Scene.wait`
- New convenience method: `Scene.pause` (alias for `Scene.wait(frozen_frame=True)`)
- Changed default behavior for OpenGL updaters: updater functions are now not called by default when they are added to the mobject
- Changed default behavior of `Scene.should_mobjects_update`: made it respect the set value of `Wait.frozen_frame`, changed automatic determination of frozen frame state to also consider Scene updaters

## Motivation and Explanation: Why and how do your changes improve the library?

This should make the behavior of updaters a little bit more consistent, both between renderers, as well as between scene and mobject updaters.

## Links to added or changed documentation pages

- [Wait](https://manimce--2504.org.readthedocs.build/en/2504/reference/manim.animation.animation.Wait.html#manim.animation.animation.Wait)
- [Scene.should_update_mobjects](https://manimce--2504.org.readthedocs.build/en/2504/reference/manim.scene.scene.Scene.html#manim.scene.scene.Scene.should_update_mobjects)
- [Scene.wait](https://manimce--2504.org.readthedocs.build/en/2504/reference/manim.scene.scene.Scene.html#manim.scene.scene.Scene.wait)
- [Scene.pause](https://manimce--2504.org.readthedocs.build/en/2504/reference/manim.scene.scene.Scene.html#manim.scene.scene.Scene.pause)

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by behackl** [comment]
The fix for the broken rendering of scenes starting with wait in OpenGL unfortunately also broke the disabled updaters again. Needs some more work.

**Comment by behackl** [comment]
Done, works, ready for review!

File: manim/scene/scene.py [hunk 1]
```diff
@@ -348,18 +348,33 @@ def update_self(self, dt):
...
+        Returns True if the mobjects of this scene should be updated.
 
-        Returns
-        -------
-            bool
+        In particular, this checks whether
+
+        - the :attr:`always_update_mobjects` attribute of :class:`.Scene`
+          is set to ``True``,
+        - the :class:`.Scene` itself has time-based updaters attached,
```
Comment (Darylgolden) [inline manim/scene/scene.py:359]: Not sure if it's in the scope of this PR, but I think it should check whether any updaters are attached, not just time-based updaters.

Comment (behackl) [reply manim/scene/scene.py:359]: Wouldn't be hard to change at all, sure! Can change it tomorrow morning.

My first thought was that I'm not sure whether this is really necessary and has a practical use case, but I don't think it'd really hurt performance or semantics to check for any present updaters.

Comment (behackl) [reply manim/scene/scene.py:359]: In light of the recent discussions on Discord, I have developed a slight preference for keeping things as they are. If users really explicitly want to enforce frame-rate dependent behavior, then (a) their updater could get a dummy `dt` argument, or (b) they could use `self.wait(frozen_frame=False)` with this PR.

Comment (Darylgolden) [reply manim/scene/scene.py:359]: SGTM

**Review by icedcoffeeee — APPROVED** [review]
Left a comment on a possibly unintentional print function. Otherwise LGTM, thank you!

File: manim/renderer/opengl_renderer.py [hunk 2]
```diff
@@ -410,10 +410,24 @@ def update_skipping_status(self):
     def play(self, scene, *args, **kwargs):
         # TODO: Handle data locking / unlocking.
         self.animation_start_time = time.time()
+        print(self.skip_animations)
```
[the remark this answers is not in the document]
Comment (behackl) [reply manim/renderer/opengl_renderer.py]: Indeed, my bad! Fixed via d0245a90

**Review by Darylgolden — APPROVED** [review]
LGTM!

**Comment by behackl** [comment]
Thanks for the reviews!

**Pull request merged**